pkg/reconciler/testing/reconcile.go

```diff
@@ -387,6 +388,10 @@ func propagateAdapterAvailabilityFunc(adapter runtime.Object) func(src v1alpha1.
 			src.GetStatusManager().PropagateDeploymentAvailability(context.Background(), a, nil)
 		case *servingv1.Service:
 			src.GetStatusManager().PropagateServiceAvailability(a)
+
+			if v1alpha1.IsMultiTenant(src) {
+				src.GetStatusManager().SetRoute(routing.URLPath(src))
+			}
 		}
```

Comment:
And here is the part where we append the URL path to the source's address (`/ns/name` instead of `/`).